ui/src/yupi18n.ts

```diff
@@ -0,0 +1,11 @@
+import { setLocale } from 'yup';
```

Comment:
Moim zdaniem średnio przydatne, bo nie zapewnia obsługi swoich pól: np. przy regex'ie wyświetla całego regexa, a nie wypadałoby pisać, że "hasło jest niepoprawne" nie wiadomo dlaczego.

Reply:
czemu średnio przydatne? Jeśli pole będzie wymagane to wolisz za każdym razem podawać jakiego klucza ma użyć? Daje to właśnie możliwość że wprowadzamy domyślne wiadomości które używają translacji, jeśli potrzebujemy użyć innej wiadomości to nie ma problemu żeby ją zdefiniować podczas tworzenia schemy.